src/stores/api.ts

```diff
@@ -140,6 +159,20 @@ export const defineAPIStore = defineStore('api', {
             });
             return types;
         },
+        buildingsByDispatchCenter: (state): Record<number, Building[]> => {
+            const dispatchCenters: Record<number, Building[]> = {};
+            state.buildings.forEach(building => {
+                const dispatchId = building.leitstelle_building_id;
+
+                // do not process dispatch centers
+                if (!dispatchId) return;
```

Comment:
same here: this won't work for buildings that aren't assigned to any dispatch center (which is possible)